staging/src/k8s.io/client-go/tools/record/event.go

```diff
@@ -255,13 +255,13 @@ func recordEvent(sink EventSink, event *v1.Event, patch []byte, updateExistingEv
 	switch err.(type) {
 	case *restclient.RequestConstructionError:
 		// We will construct the request the same next time, so don't keep trying.
-		klog.Errorf("Unable to construct event '%#v': '%v' (will not retry!)", event, err)
+		klog.Errorf("Unable to construct event '%#v, Message: %s': '%v' (will not retry!)", event.InvolvedObject, event.Message, err)
```

Comment:
if we fail to construct the event, it seems like all the fields would be pertinent, so I would expect to see them continue to be rendered.

Reply:
ok